docs/docs-network/operation/sequencer_management/claiming_rewards.md

```diff
@@ -43,121 +45,95 @@ Before claiming, verify these conditions:
-# Output: 0.1
-```
+If the result shows `100000000000000000` (decimal), that equals `0.1` tokens.
 
 ## Claiming Your Rewards
 
 The `claimSequencerRewards` function is permissionless - anyone can call it for any address. Rewards are always sent to the `coinbase` address, regardless of who submits the transaction.
 
-### Basic Claim Command
+### Claim via Etherscan
 
-Use `cast send` to claim rewards:
+1. Go to your Rollup contract's **Write Contract** page:
+   - **Mainnet**: `https://etherscan.io/address/[ROLLUP_ADDRESS]#writeContract`
+   - **Testnet (Sepolia)**: `https://sepolia.etherscan.io/address/[ROLLUP_ADDRESS]#writeContract`
 
-```bash
-cast send $ROLLUP_ADDRESS \
-  "claimSequencerRewards(address)" \
-  [COINBASE_ADDRESS] \
-  --rpc-url $RPC_URL \
-  --private-key [YOUR_PRIVATE_KEY]
-```
+2. Click **"Connect to Web3"** and connect your wallet (MetaMask, WalletConnect, etc.)
 
-Replace:
-- `[COINBASE_ADDRESS]` - The coinbase address whose rewards you want to claim
-- `[YOUR_PRIVATE_KEY]` - The private key of the account paying for gas
-
-**Example:**
-```bash
-cast send $ROLLUP_ADDRESS \
-  "claimSequencerRewards(address)" \
-  0x742d35Cc6634C0532925a3b844Bc9e7595f0bEb \
-  --rpc-url $RPC_URL \
-  --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
-```
+3. Find the `claimSequencerRewards` function
 
-### Using a Keystore File
+4. Enter the coinbase address whose rewards you want to claim (e.g., `0x742d35Cc6634C0532925a3b844Bc9e7595f0bEb`)
 
-For better security, use a keystore file instead of exposing your private key:
+5. Click **"Write"**
 
-```bash
-cast send $ROLLUP_ADDRESS \
-  "claimSequencerRewards(address)" \
-  [COINBASE_ADDRESS] \
-  --rpc-url $RPC_URL \
-  --keystore [PATH_TO_KEYSTORE] \
-  --password [KEYSTORE_PASSWORD]
-```
+6. Confirm the transaction in your wallet
 
-### Using a Hardware Wallet
+### Using Hardware Wallets
 
-If you're using a Ledger wallet:
+Etherscan supports hardware wallet connections through:
 
-```bash
-cast send $ROLLUP_ADDRESS \
-  "claimSequencerRewards(address)" \
-  [COINBASE_ADDRESS] \
-  --rpc-url $RPC_URL \
-  --ledger
-```
+- **MetaMask + Ledger/Trezor**: Connect your hardware wallet to MetaMask, then use MetaMask to connect to Etherscan
+- **WalletConnect**: Many hardware wallet apps support WalletConnect for direct connection
 
-This will prompt you to confirm the transaction on your Ledger device.
+This ensures your private keys never leave your hardware device.
 
 ## Verifying Your Claim
 
-Check that the transaction succeeded and your pending rewards were reset to zero:
+### Check Transaction Status
+
+After submitting the claim transaction:
+
+1. Copy the transaction hash from your wallet or Etherscan
+2. Go to `https://etherscan.io/tx/[TRANSACTION_HASH]` (use `sepolia.etherscan.io` for testnet)
+3. Verify the transaction status shows **"Success"**
+
+### Verify Rewards Are Claimed
```

Comment:
Also not necessary, just say something along the lines of "Once the transaction has been mined, you'll find the $AZTEC tokens in your wallet"